### PR TITLE
[BLOG] Add quansight-at-scipy2019

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-quansight-at-scipy2019-quansight.vercel.app/blog/quansight-at-scipy2019).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/quansight-at-scipy2019.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: 'Quansight presence at SciPy'19'
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-at-scipy2019.md:2]:
```suggestion
title: "Quansight presence at SciPy'19"
```

File: apps/labs/posts/quansight-at-scipy2019.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Quansight presence at SciPy'19"
+published: July 15, 2019
+author: Ralf Gommers
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-at-scipy2019.md:4]:
```suggestion
author: ralf-gommers
```

File: apps/labs/posts/quansight-at-scipy2019.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Quansight presence at SciPy'19"
+published: July 15, 2019
+author: ralf-gommers
+description: 'This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed.'
+category: [Community, PyData ecosystem]
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-at-scipy2019.md:6]:
```suggestion
category: [Community, PyData Ecosystem]
```

Comment (MarsBarLee) [reply apps/labs/posts/quansight-at-scipy2019.md:6]: Update category

File: apps/labs/posts/quansight-at-scipy2019.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Quansight presence at SciPy'19"
+published: July 15, 2019
+author: ralf-gommers
+description: 'This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed.'
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-at-scipy2019.md:5]:
```suggestion
description: "This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed."
```

File: apps/labs/posts/quansight-at-scipy2019.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Quansight presence at SciPy'19"
+published: July 15, 2019
+author: ralf-gommers
+description: "This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed."
+category: [Community, PyData Ecosystem]
+  src: /posts/quansight-at-scipy2019/blog_feature_org.svg
```
Comment (MarsBarLee) [inline apps/labs/posts/quansight-at-scipy2019.md:7]:
```suggestion
featuredImage:
  src: /posts/quansight-at-scipy2019/blog_feature_org.svg
```

**Review by noatamir — APPROVED** [review]
LGTM. merging.

**Pull request merged**